ci/pipelines/toolbox-staging-deploy.yml

```diff
@@ -78,17 +93,8 @@ jobs:
             path: /bin/sh
             args:
             - -ec
-            - | 
+            - |
               DIGEST=$(cat toolbox-ecr-registry-staging/digest)
```

Comment:
you could load this digest as a var as well if you want to be really nifty (see how we load the tag in deploy job)

Reply:
Done